### ISSUE TITLE
The protein group number starts from 2 in Philosopher 1.5.2

### ISSUE THREAD

**Issue by fcyu** [Nesvilab/philosopher#81]
Hi Felipe,

I guess it is supposed to start from 1, right? The data and setting are the same as that from issue [https://github.com/Nesvilab/philosopher/issues/80](https://github.com/Nesvilab/philosopher/issues/80).

Here is the protein.tsv.
[protein.zip](https://github.com/Nesvilab/philosopher/files/3747952/protein.zip)

Could you please take a look when you have time?

Thanks,

Fengchao

**Comment by anesvi** [comment]
Encrypted Message Attached from Michigan Medicine.

You are receiving this secure, encrypted email message because it may contain sensitive information. 

If you have concerns about the validity of this message, contact the sender directly and ensure the email address is a known @med.umich.edu email address.

To read your message, open the attachment: securedoc.html

•	You will be prompted to open (view) the file or save (download) it to your computer. For best results, save the file first, then open it in a Web browser. No special software is required.

•	To access from a mobile device, forward this message to mobile@res.cisco.com to receive a mobile login URL. 

Additional Information:

•	Secure Email Help https://res.cisco.com/websafe/help?topic=FAQ

**Comment by prvst** [comment]
not necessarily, the group numbers and subgroups are assigned by ProteinProphet, only the groups that pass the FDR filtering goes to the final report.
